In file included from muse/liste/listedit.cpp:37:
muse/components/editevent.h:26:10: fatal error: 'ui_editnotedialogbase.h' file not found
#include "ui_editnotedialogbase.h"
         ^~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.

--- muse/liste/CMakeLists.txt.orig	2021-04-08 11:45:21 UTC
+++ muse/liste/CMakeLists.txt
@@ -43,6 +43,8 @@ add_library ( liste ${MODULES_BUILD}
       ${liste_mocs}
       )
 
+add_dependencies(liste components)
+
 ##
 ## Append to the list of translations
 ##
